example_sns_init.yaml

```diff
@@ -25,104 +31,307 @@ description: >
+
+    # The symbol of the token issued by the SNS Ledger.
+    # Must be a string with a length between 3 and 10 characters.
+    #
+    # Example: ICP
     symbol: ROT
-    transaction_fee: 10_000 e8s # You can leave this alone.
+
+    # SNS ledger transactions will have this fee.
+    transaction_fee: 10_000 e8s
+
+    # Path to the SNS token logo on your local filesystem. The path is relative
+    # to the configuration file location, unless an absolute path is given.
+    # Must have less than 341,334 characters, roughly 256 Kb. The only supported
+    # format is PNG.
     logo: logo.png
 
-# It is fine to leave these alone.
+# Configures SNS proposal-related fields. These fields have no
+# effect on the NNS proposal that is created from this file, but will affect all
+# SNS proposals submitted to the newly created SNS.
 Proposals:
+    # The cost of making an SNS proposal that is rejected by the SNS Neuron
+    # holders.
     rejection_fee: 1 token
+
+    # The initial voting period of a newly-created SNS proposal. (A proposal's
+    # voting period may be increased during its lifecycle due to the
+    # wait-for-quiet algorithm (see details below).
     initial_voting_period: 4d
+
+    # The wait for quiet algorithm extends the voting period of a proposal when
+    # there is a flip in the majority vote during the proposal's voting period.
+    #
+    # Without this, there could be an incentive to vote right at the end of a
+    # proposal's voting period, in order to reduce the chance that people will
+    # see and have time to react to that.
+    #
+    # TODO: fix text below
+    # If this value is set to 1 day, then a change in the majority vote at the
+    # end of a proposal's original voting period results in being extended by an
+    # additional day. Another change at the end of the extended
+    # period will cause the voting period to be extended by half a day, and so on.
+    #
+    # The total extension to the voting period will never be more than twice this
+    # value.
+    #
+    # For more information, please refer to
+    # https://wiki.internetcomputer.org/wiki/Network_Nervous_System#Proposal_decision_and_wait-for-quiet
     maximum_wait_for_quiet_deadline_extension: 1 day
 
-# It is fine to leave these alone.
+# Configuration of SNS neuron-related fields.
 Neurons:
+    # The minimum amount of SNS tokens an SNS ledger account must have to stake
+    # a neuron. The larger the minimum, the more tokens must be acquired to be
+    # able to vote.
     minimum_creation_stake: 1 tokens
 
-# It is fine to leave these alone.
+# Configuration of voting-related fields of the SNS. These fields have no effect
+# on the NNS proposal created by this file, but will affect the SNS itself.
 Voting:
+    # The minimum dissolve delay a neuron must have to be able to cast votes on
+    # proposals. The higher the minimum, the longer users will need to be
+    # aligned with the SNS to vote. However, the longer the minimum, the longer
```

Comment:
I find the word "aligned" misleading: it sounds like you must vote in a particular way.

Reply:
Agree, but I couldn't easily rephrase it so far.